### DISCUSSION TITLE
Question about settings modal

### DISCUSSION THREAD

**Discussion by lgalloherran** [twilio-labs/paste#1429] [Q&A] [answered]
Hi Dsys, 
I'm redesigning the 'Sender selection settings' (which currently work in a disclosure format and switch format)  for this page and I have this option: 
https://www.figma.com/proto/kem6iZab8XFhLo8S0Gx0AA/PHONE-NUMBERS-Manage-Sender-Ids-section?page-id=64%3A30055&node-id=64%3A30056&viewport=490%2C225%2C0.2335478514432907&scaling=min-zoom

I want to know if this is option possible from a accessibility and system perspective, or if there is a better one (tab maybe?)

**Comment by TheSisb** [comment, marked as answer]
Hi @lgalloherran,

The placement of the `Sender selection settings` button is good, and the button is the correct UI to trigger the modal. For the modal, we don't currently have a Switch component, it's in our roadmap for the 2021.10.05 release. An alternative before then may be to use the checkbox component and add a save button to the footer of the modal, which helps to give the modal a more actionable feel.

If I missed anything, please follow up :)
